webrender/src/display_list_flattener.rs

```diff
@@ -56,8 +56,8 @@ pub struct ClipIdToIndexMapper {
 
 impl ClipIdToIndexMapper {
     pub fn add_clip_chain(&mut self, id: ClipId, index: ClipChainIndex) {
-        debug_assert!(!self.clip_chain_map.contains_key(&id));
-        self.clip_chain_map.insert(id, index);
+        let _is_new_key = self.clip_chain_map.insert(id, index).is_none();
```

Comment:
nit: let's not confuse the reader with `is_none()` here and instead move it inside the assert

Reply:
Fixed